README.md

```diff
@@ -1,16 +1,18 @@
 # teamcity-client
 
 ## Introduction
-Teamcity client is a simple scala library which wraps REST calls to the Teamcity REST API. Use this library to perform different operations and get data from the Teamcity server. This version is tested with Teamcity version **2018.1.5 (build 58744)**. This library can be used also with Java but will require to use the **scala.collection.JavaConverters** when using the collections (see in the examples below).
+Teamcity client is a wrapper for Teamcity REST API calls for scala. It is a simple library that makes it easy to perform different operations and get data from the Teamcity server.
 
-Teamcity API returns only a small set of properties of the entity when getting a collection of entities, we mapped these to Base{Entity}. You can use the id returned in Base{Entity} to retrive the full properties of the entity (by calling get{Entity}ById methods).
+Current version is tested with Teamcity version **2018.1.5 (build 58744)**. Teamcity client can also be used with Java but will require **scala.collection.JavaConverters** when using collections (see examples below).
 
-TeamCity API collection entities conain a count property and a list property, The list property name is in single not plural, In order to stay as close to the API as possible, Teamcity client follows this convention. 
+Since Teamcity API only returns a small set of properties for each entity when getting a collection of entities, so we mapped these to `Base{Entity}`. You can use the `id` returned in `Base{Entity}` to retrieve the full properties of any select entity (by calling `get{Entity}ById` methods).
 
-Please note that not all the API calls and objects are covered in this TeamCityClient.
+The collection entities of TeamCity API contain a count property and a list property, and the list property name is in singular, not plural form - in order to stay as close to the API as possible, Teamcity client also follows this convention.
+
+Please note that not all API calls and objects are covered in this README.
```

Comment:
not in the readme but in the client, there are some apis and entities which we didnt cover in the client

Reply:
So the client does not use / wrap around every single TeamCity API endpoint, just some of the most important ones (or something like that), correct?

Reply:
correct, i wouldnt add the important ons as this is subjective to the application using the client

Reply:
Should be resolved now, please check.